src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +96,7 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={!isFormValid()}
```

Comment:
is there a reason to have it as a function?
```suggestion
            disabled={!isFormValid}
```

Reply:
actually no, good point. probably because i have been heavily involved in writing functions recently 😄